as-vending/main.go

```diff
@@ -70,40 +72,40 @@ func main() {
 
 	var err error
 
-	err = edgexSdk.AddRoute("/boardStatus", vendingState.BoardStatus, "POST")
-	errorAddRouteHandler(edgexSdk, err)
+	err = service.AddRoute("/boardStatus", vendingState.BoardStatus, "POST")
+	errorAddRouteHandler(lc, err)
 
-	err = edgexSdk.AddRoute("/resetDoorLock", vendingState.ResetDoorLock, "POST")
-	errorAddRouteHandler(edgexSdk, err)
+	err = service.AddRoute("/resetDoorLock", vendingState.ResetDoorLock, "POST")
+	errorAddRouteHandler(lc, err)
 
-	err = edgexSdk.AddRoute("/maintenanceMode", vendingState.GetMaintenanceMode, "GET", "OPTIONS")
-	errorAddRouteHandler(edgexSdk, err)
+	err = service.AddRoute("/maintenanceMode", vendingState.GetMaintenanceMode, "GET", "OPTIONS")
+	errorAddRouteHandler(lc, err)
 
 	// create the function pipeline to run when an event is read on the device channels
-	err = edgexSdk.SetFunctionsPipeline(
-		transforms.NewFilter(vendingState.Configuration.DeviceNames).FilterByDeviceName,
+	err = service.SetFunctionsPipeline(
+		transforms.NewFilterFor(vendingState.Configuration.DeviceNames).FilterByDeviceName,
```

Comment:
I don't see Configuration.DeviceNames in the TOML. Where is this coming from?

Reply:
it is in second layer: ServiceConfiguration.DeviceNames

Reply:
Is that a constant?

Reply:
I think so, here it is in the configuration.toml: DeviceNames   = "ds-card-reader,Inference-MQTT-device"